src/tpt_reports/tpt_reports.py

```diff
@@ -73,15 +161,29 @@ def main() -> None:
 
     # Assign validated arguments to variables
     log_level: str = validated_args["--log-level"]
+    servicenow_id: str = validated_args["SERVICENOW_ID"]
+    election_name: str = validated_args["ELECTION_NAME"]
+    domain_tested: str = validated_args["DOMAIN_TESTED"]
+    output_directory: str = validated_args["OUTPUT_DIRECTORY"]
     json_file_path: str = validated_args["JSON_FILE_PATH"]
 
     # Set up logging
     logging.basicConfig(
         format="%(asctime)-15s %(levelname)s %(message)s", level=log_level.upper()
     )
 
-    if load_json_file(json_file_path):
-        LOGGER.info("JSON FILE loaded successfully.")
+    LOGGER.info("Loading TPT Report, Version : %s", __version__)
+
+    # Issue #27 - Input and output file directories change
+    # TODO: Validate that the output_directory is not in the repo.
+    # Create output directory
+    if not os.path.exists(validated_args["OUTPUT_DIRECTORY"]):
+        os.mkdir(validated_args["OUTPUT_DIRECTORY"])
+
+    if generate_reports(
+        servicenow_id, election_name, domain_tested, output_directory, json_file_path
+    ):
+        LOGGER.info("Report generated successfully.")
```

Comment:
I recommend including some identifying data (e.g. servicenow_id, election_name, domain_tested) in the log message here, as well as adding a corresponding message before the report generation begins.  If you generate multiple reports consecutively, you will most likely want to be able to tell when each report started and finished.

Reply:
We have discussed this, and we have an issue to address logging report generation beginning/ending and a more descriptive filename in Issue #28. TODO comments have been added in : 883d44cc9fa473e28c0f80e809b972570a13cfce